CMAKE error when following instructions in README.txt

even when specifying `CMAKE_INSTALL_PREFIX`, i get this error:
```
CMake Error at CMakeLists.txt:151 (install):
  install TARGETS given no LIBRARY DESTINATION for shared library target
  "sxg".
```
including a `CMAKE_INSTALL_LIBDIR` fixes, so it looks like the documentation change made in https://github.com/google/libsxg/commit/1559f01723a0c0c31ccf3951ce09d3f9a09bc532 is innacturate.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -61,12 +61,10 @@ if (NOT CMAKE_BUILD_TYPE)
   set (CMAKE_BUILD_TYPE "Debug")
 endif ()
 
-if (RUN_TEST)
-  # `target_link_options` used in fuzzing test requires 3.13
-  cmake_minimum_required (VERSION 3.13)
-else ()
-  cmake_minimum_required (VERSION 3.0)
-endif ()
+# When RUN_TEST is enabled, `target_link_options` used in fuzzing test requires 3.13.
+# When RUN_TEST is disabled, I'm not sure what minimum version is required, but
+# anecdotally it's higher than 3.0.
+cmake_minimum_required (VERSION 3.13)
 
 set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wextra -std=c++11 -fPIC")
 set (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall -Wextra -std=c99 -fPIC -D_POSIX_C_SOURCE=200112L")

